CMakeLists.txt

```diff
@@ -0,0 +1,225 @@
+
+cmake_minimum_required(VERSION 3.4.3)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
+
+project(dispatch C CXX)
+enable_testing()
+
+set(CMAKE_C_VISIBILITY_PRESET hidden)
+set(CMAKE_CXX_STANDARD 11)
+
+set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
+set(THREADS_PREFER_PTHREAD_FLAG TRUE)
+find_package(Threads REQUIRED)
+
+include(GNUInstallDirs)
+include(ExternalProject)
+if(BUILD_SHARED_LIBS)
+  set(CMAKE_LIBRARY_TYPE SHARED)
+else()
+  set(CMAKE_LIBRARY_TYPE STATIC)
+endif()
+
+set(WITH_KQUEUE "" CACHE PATH "Path to kqueue headers")
```

Comment:
I think KQUEUE has been removed in https://github.com/apple/swift-corelibs-libdispatch/pull/215/files?